### PR TITLE
WIP : Add zotero URI handling

### LINKED ISSUE #64: Missing x-scheme-handler directive from the desktop file (zotero:// URL schemes don't work)
I think the desktop file should be patched with the `MimeType=x-scheme-handler/zotero;` directive to enable the zotero:// URL scheme. 

More info: https://forums.zotero.org/discussion/78550/getting-zotero-to-work-under-ubuntu-linux-solved

### PR DESCRIPTION
**Pull request by lionirdeadman**
Fix #64

### CODE REVIEW

**Comment by felipehw** [comment]
It works?

**Comment by lionirdeadman** [comment]
No, it seems that while it opens the program, for some reason the application doesn't open the file that the link points to.

**Comment by felipehw** [comment]
:(

**Comment by pbsds** [comment]
I think the `Exec` entry in the .desktop file is missing `-url %U`.
Example: https://aur.archlinux.org/cgit/aur.git/tree/zotero.desktop?h=zotero

File: org.zotero.Zotero.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
         "install -D chrome/icons/default/default32.png /app/share/icons/hicolor/32x32/apps/zotero.png",
         "install -D chrome/icons/default/default48.png /app/share/icons/hicolor/48x48/apps/zotero.png",
         "install -D chrome/icons/default/default256.png /app/share/icons/hicolor/256x256/apps/zotero.png",
-        "desktop-file-install --dir=/app/share/applications --set-key=Exec --set-value=zotero --set-key=Icon --set-value=org.zotero.Zotero zotero.desktop",
+        "desktop-file-install --dir=/app/share/applications --set-key=Exec --set-value='zotero %U' --set-key=Icon --set-value=org.zotero.Zotero zotero.desktop --add-mime-type=x-scheme-handler/zotero",
```
[the remark this answers is not in the document]
Comment (felipehw) [reply org.zotero.Zotero.json:56]: I tested it here. It works.

**Comment by felipehw** [comment]
Many thanks for fixing this issue.

I don't know how to add the `build.sh` changes here (neither how to incorporate the suggestions of @wshanks).

So, I'm doing a new one with the original changes at `org.zotero.Zotero.json` and the needed changes at `build.sh`: https://github.com/flathub/org.zotero.Zotero/pull/91.

Please, let's always update the `build.sh` instead directly the `org.zotero.Zotero.json` and `org.zotero.Zotero.appdata.xml`. This helps to keep the Zotero Flatpak updated with minimal efforts.

File: org.zotero.Zotero.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (felipehw) [reply org.zotero.Zotero.json:56]: I tested it here. It works.

**Comment by wshanks** [comment]
Sorry, I missed the build.sh. By the way, for keeping the flatpak updated, you might look at https://github.com/flathub/flatpak-external-data-checker. I use it on the flatpak I maintain and it has worked well (the `x-data-checker` entries in [this yaml](https://github.com/flathub/de.klayout.KLayout/blob/master/de.klayout.KLayout.yaml)).

**Pull request closed without merging**